### PR TITLE
Reduce the number of logs sent from timing functions

### PR DESCRIPTION
**Pull request by Atala**

### CODE REVIEW

File: app/config/config_prod.yml [hunk 1]
```diff
@@ -83,7 +83,7 @@ monolog:
         timing:
             type: rotating_file
             path: "%env(APP_LOG_DIR)%/timing.%kernel.environment%.log"
-            level: debug
+            level: info
```
Comment (vladimir-8) [inline app/config/config_prod.yml:86]: This setting does not affect Datadog. It's only for local files. So we might keep it on `debug` however these logs might take a lot of disk space at some point 😁

Comment (Atala) [reply app/config/config_prod.yml:86]: yes it was i understood i saw only INFO logs in datado :+1:

**Comment by Atala** [comment]
@vladimir-8 where are these changes ? in a branch or in master ?

**Comment by vladimir-8** [comment]
They were in `master`, but I guess some log levels were reverted during rebase/merging

**Pull request merged**